### PR TITLE
feat(igxGrid): Add Input for notification snackbar display time.

### LINKED ISSUE #8398: "Row added" snackbar is not visible if the grid has small height.
## Description  
When grid has no height and no data and a single row is added the "Row added" snackbar hide behind the grid header.

 * igniteui-angular version: 
 * browser: 

## Steps to reproduce  

1. Open the gridAddRow dev sample.
2. Remove the grid height.
3. Add row via the button.

image

## Result  
Snackbar hides behind header and is only partially visible.

## Expected result  
Snackbar should be visible.

## Attachments  
Attach a sample if available, and screenshots, if applicable.

### LINKED ISSUE #8399: "Row added" snackbar hides too fast.
## Description  
Currently the "Row Added" snackbar, that notifies when a row is added and allows scrolling to it via the "Show" button is configured to hide after 2000ms.
For some users that may be too soon as they may not be able to react in time to click the "Show" button and after the snackbar hides there's no way for them to find the added row.

Possible solutions are either to set a bigger default display time or expose the display time as an Input so that users can adjust it to their liking.

### PR DESCRIPTION
**Pull request by MayaKirova**
Closes #8399

### Additional information (check all that apply):
 - [ ] Bug fix
 - [x] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [x] This PR includes unit tests covering all the new code ([test guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Test-implementation-guidelines-for-Ignite-UI-for-Angular))
 - [ ] This PR includes API docs for newly added methods/properties ([api docs guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Documentation-Guidelines))
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes ([migrations guidelines](https://github.com/IgniteUI/igniteui-angular/wiki/Update-Migrations))
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

**Review by dkamburov — DISMISSED** [review]

File: projects/igniteui-angular/src/lib/grids/grid-base.directive.ts [hunk 1]
```diff
@@ -250,6 +245,14 @@ export abstract class IgxGridBaseDirective extends DisplayDensityBase implements
         return this._resourceStrings;
     }
 
+    /**
+     * Gets/Sets the display time for the row adding snackbar notification.
+     * @remarks
+     * By default it is 2000ms.
+     */
+    @Input()
+    public snackbarDisplayTime = 2000;
```
Comment (kdinev) [inline projects/igniteui-angular/src/lib/grids/grid-base.directive.ts:254]: There should definitely be an input for this, but the default is way too short. Until the user reads the info it's gone. Please bump the default up to 5000 or 7000 (test which is better).

**Review by kdinev — CHANGES_REQUESTED** [review]

File: projects/igniteui-angular/src/lib/grids/grid-base.directive.ts (hunk 1 above)
Comment (ChronosSF) [reply projects/igniteui-angular/src/lib/grids/grid-base.directive.ts:254]: This default is based on @StefanIvanov 's decision. We can do it as long as you guys agree on.

Comment (kdinev) [reply projects/igniteui-angular/src/lib/grids/grid-base.directive.ts:254]: 2 seconds is way too short to read and react. @StefanIvanov give your opinion please.

Comment (StefanIvanov) [reply projects/igniteui-angular/src/lib/grids/grid-base.directive.ts:254]: I am fine with changing the default value to anything below 10000 so let's bump it up to 6000

**Review by kdinev — APPROVED** [review]

**Review by StefanIvanov — APPROVED** [review]

**Pull request merged**